onedal/linear_model/linear_model.py

```diff
@@ -198,8 +161,43 @@ def __init__(
         super().__init__(fit_intercept=fit_intercept, copy_X=copy_X, algorithm=algorithm)
 
     def fit(self, X, y, queue=None):
-        return super()._fit(X, y, _backend.linear_model.regression, queue)
```

Comment:
please leave it as is here, do not move _fit impl from base class

Reply:
base class should contain only methods which are used in both LinearRegression and IncrementalLinearRegression. in my understanding it is designed exactly for this purpose. _fit method is not of this type. so, _fit method should be moved to LinearRegression class. LinearRegression class already contains method fit which only calls _fit method and nothing else and _fit method is not called anywhere else. in this situation I am completely sure that _fit method is redundant and its content can be safely moved to fit method to increase code readability.